java/ql/lib/semmle/code/java/security/SensitiveActions.qll

```diff
@@ -14,13 +14,19 @@
 import java
 
 private string suspicious() {
-  result = ["%password%", "%passwd%", "%account%", "%accnt%", "%trusted%"]
+  result =
+    [
+      "%password%", "%passwd%", "pwd", "%account%", "%accnt%", "%trusted%", "%refresh%token",
+      "%secret%token"
+    ]
 }
 
 private string nonSuspicious() {
   result = "%hashed%" or
   result = "%encrypted%" or
-  result = "%crypt%"
+  result = "%crypt%" or
+  result = "%create table%" or
+  result = "%drop table%"
```

Comment:
That's a little surprising -- `drop table` + userSuppliedString sounds like the worst case scenario :) Add a comment explaining why you're marking `drop table` as safe?

Reply:
This isn't actually user input but rather potentially sensitive (or non-sensitive in this case) values, no matter where they come from.

My intention with this was discarding things like `CREATE TABLE passwords` because that doesn't contain sensitive values, but your comment actually made me realize `nonSuspicious` is looking at variable and method _names_, not literal string values, so those two lines are not necessary. I confirmed that by removing them in 10e5f96 and checking that the tests still pass, specially [this test case](https://github.com/github/codeql/pull/6492/files#diff-a1567e122cbcdd1a11942a76e32daa54a435e183bd4c9262fa9ffe381222d836R17).